docs/source/using/managing-minishift.adoc

```diff
@@ -293,23 +293,32 @@ CONTAINER    IMAGE                   COMMAND                CREATED        STATU
 71fe8ff16548 openshift/origin:v1.5.1 "/usr/bin/openshift s" 4 minutes ago  Up 4 minutes  origin
 ----
 
-[[experimental-functionality]]
-== Experimental functionality
+[[enabling-experimental-openshift-features]]
+== Enabling Experimental OpenShift Features
 
-To allow users and developers to get early access to some upcoming features, we offer a way to enable experimental functionality.
+By default, Minishift does not expose all `oc cluster up` flags in the CLI, either because certain features are not considered stable or because Minishift does not yet support the latest options in `oc cluster up`.
```

Comment:
I recall that we had this before, but decided it was to specific. This description is geared towards _oc_ and _cluster up_ flags. This is what we use it for now, but experimental features could and probably will be used for other things as well. For example @gbraad is looking into adding options for provisioning Hyper-V against a fixed IP. This might go behind the experimental flag as well, but has nothing to do with _cluster up_.